libp2p/muxers/mplex/lpchannel.nim

```diff
@@ -64,16 +65,16 @@ type
 
 func shortLog*(s: LPChannel): auto =
   try:
-    if s.isNil: "LPChannel(nil)"
+    if s == nil: "LPChannel(nil)"
     elif s.name != $s.oid and s.name.len > 0:
       &"{shortLog(s.conn.peerId)}:{s.oid}:{s.name}"
     else: &"{shortLog(s.conn.peerId)}:{s.oid}"
   except ValueError as exc:
-    raise newException(Defect, exc.msg)
+    raiseAssert(exc.msg)
```

Comment:
@etan-status should we be raising a Defect here?

Reply:
`raiseAssert` raises a `Defect` :)